www/templates/www/user.html

```diff
@@ -76,43 +76,13 @@ <h2>{% trans 'Service subscriptions' %}</h2>
               {% blocktrans with ref=subscription.reference_number cost=subscription.service.cost_string %}This service is paid with reference number <strong>{{ ref }}</strong>
                 to bank account <strong>{{bank_iban}}</strong> (recurring monthly payments). <br />Service price is <strong>{{ cost }}</strong>.{% endblocktrans %}<br/>
             </p>
-            <p>{% blocktrans %}To pay for multiple months at once, you can create a custom invoice. {% endblocktrans %}
-              <a href="{% url 'custominvoice' %}">{% trans 'Create and view custom invoices' %}</a>
-            </p>
             {% endif %}
           {% endif %}
         </td>
       </tr>
     </table>
     {% endfor %}
-    <h2>{% trans 'Custom invoices' %}</h2>
-    <table class="table table-responsive-sm">
-      <tr>
-        <th>{% trans 'Date' %}</th>
-        <th>{% trans 'Service' %}</th>
-        <th>{% trans 'Amount' %}</th>
-        <th>{% trans 'Reference number' %}</th>
-        <th>{% trans 'Paid' %}</th>
-      </tr>
-      {% for custominvoice in userdetails.custominvoices %}
-      <tr>
-        <td>{{ custominvoice.created }}</td>
-        <td>{{ custominvoice.days }} {% trans 'days of' %} {{ custominvoice.subscription.service.name }}</td>
-        <td>{{ custominvoice.amount }}</td>
-        <td>{{ custominvoice.reference_number }}</td>
-        <td>
-          {% if custominvoice.payment_transaction %}
-            <a href="{% url 'banktransaction-view' custominvoice.payment_transaction.id %}">{{ custominvoice.payment_transaction.date }}</a>
-          {% else %}
-            {% trans 'Not paid yet' %}
-          {% endif %}
-        </td>
-      </tr>
-      {% endfor %}
-    </table>
-    <p>
-      <a href="{% url 'custominvoice' %}">{% trans 'Create and view custom invoices' %}</a>
-    </p>
+
```

Comment:
I had this in my mind: https://github.com/TampereHacklab/mulysa/compare/hide_custom_invoices_a_bit...sbeach92:mulysa:mulysa-ui#diff-798da5eaf7fbe1c806b2857557d68d81efa4b2e06dc6a1f1bac42af647d98c8aL88-R136
Which would result this without custom invoices:
![image](https://github.com/TampereHacklab/mulysa/assets/75908489/ce89e7c0-dcaa-4962-9a30-70dcee8658aa)
And this with custom invoices:
![image](https://github.com/TampereHacklab/mulysa/assets/75908489/8637bcfd-5d00-488f-9963-f3fdb07dc245)

Reply:
made the strikeout a bit differently, but now it does the whole tr strikeout (and the class is reusable and should work pretty much everywhere) [eec5767](https://github.com/TampereHacklab/mulysa/pull/480/commits/eec57679d7a1e156302056da4d9cf301d8be8bd4)